packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -2324,14 +2324,42 @@ class WebpackCLI {
       return;
     }
 
+    let needForceShutdown = false;
+    if (compiler.options.cache) {
```

Comment:
Please change it on:
```
const needGracefullShutdown = (compiler) =>
      compiler.compilers
        ? compiler.compilers.some((compiler) => compiler.options.cache)
        : compiler.options.cache;
```

Reply:
`needGracefullShutdown` is semaphore for signals, but I got what change you want :)

Reply:
marking it as resolved